nightwatch.conf.js

```diff
@@ -0,0 +1,11 @@
+const path = require('path')
+const PHOENIX_PATH = process.env.PHOENIX_PATH
+
+const config = require(path.join(PHOENIX_PATH, "nightwatch.conf.js"))
+
+config.page_objects_path = [PHOENIX_PATH + `/tests/acceptance/pageObjects`, 'ui/tests/acceptance/pageobjects']
+config.custom_commands_path = PHOENIX_PATH + `/tests/acceptance/customCommands`
+
+module.exports = {
+    ...config
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces but found 4.](https://app.codacy.com/gh/owncloud/ocis-hello/pullRequest?prid=5709816)